File: BFF/app/models/product.php
<?php

    class Product{
        private int $id;
        private string $name;
        private string $archieve;
        private int $quantity;
        private string $description;
        private string $type;
        private int $price;
        
        public function __construct(
            int $id,
            string $name,
            string $archieve,
            int $quantity,
            string $description,
            string $type,
            int $price
            ){
            $this->id = $id;
            $this->name = $name;
            $this->archieve = $archieve;
            $this->quantity = $quantity;
            $this->description = $description;
            $this->type = $type;
            $this->price = $price;
        }
    }

?>

File: Back-end/src/main/java/com/alpha/studio/code/university/project/domain/model/Product.java
package com.alpha.studio.code.university.project.domain.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Product {

    private int id;
    private String name;
    private String archieve;
    private int quantity;
    private String description;
    private String type;
    private double price;

}
